libsam.c

```diff
@@ -349,7 +349,6 @@ struct sid_array *sam_make_sid_array(struct sid_binary *sidbuf, int size)
 
 void sam_free_sid_array(struct sid_array *array)
 {
-
```

Comment:
If everyone function has an additional line you shouldn't remove it.
I haven't checked it.
Otherwise if removing this line matches the other functions style then this typo fixing need to go into its own commit.
(I mean all the typos fixing need to go into a single commit.)